data/json/items/comestibles.json

```diff
@@ -3203,8 +3203,8 @@
         "price" : 0, "//" : "Not for use in edible/foodsafe recipes.  Inefficiency is handled by tainted tallow recipe requiring 200% as much materials input as regular tallow recipe.",
         "material" : "junk",
```

Comment:
Not changed here but: other fats have flesh material, this one has junk meaning its edible by herbivores